docker-build.sh

```diff
@@ -19,6 +19,9 @@ build_docker=0
 # set flags
 FORCE_BUILD=${FORCE_BUILD:-0}
 
+# DOCKER REPO
+DOCKER_REPO="docker.elastic.co/elasticsearch/"
```

Comment:
- Consider moving DOCKER_REPO assignment to the ver_maj case statement, below, to make it easier to maintain.